src/render/state.js

```diff
@@ -10,19 +10,17 @@ const {
  *
  * @param      {Function|Object}  newState  An object containing new keys/values
  *                                          or a function that will provide the same.
- * @param      {Function}         cb        A callback that may be provided to setState.
```

Comment:
I'm not sure I understand the change you're making here.  `setState` accepts a second option argument of callback for after the state has been updated.